### DISCUSSION TITLE
Config missing upon installation

### DISCUSSION THREAD

**Discussion by DuckersMcQuack** [prometheus/prometheus#12385] [Q&A]
### What did you do?

Install newest available version

### What did you expect to see?

It starting and running properly.

### What did you see instead? Under which circumstances?

Nothing started, and logs shows this
![image](https://github.com/prometheus/prometheus/assets/9039370/35473b30-4a95-4318-a3ed-2378263a5907)

### System information

Unraid 6.11.5

### Prometheus version

```text
It doesn't create/add a config file, so it cannot start up.
```

### Prometheus configuration file

```yaml
It's what it's missing.
```

### Logs

```text
ts=2023-05-17T19:18:09.220Z caller=main.go:479 level=error msg="Error loading config (--config.file=/etc/prometheus/prometheus.yml)" file=/etc/prometheus/prometheus.yml err="open /etc/prometheus/prometheus.yml: no such file or directory"
```

**Comment by roidelapluie** [comment]
Prometheus does not create a config file, you have to provide it.
